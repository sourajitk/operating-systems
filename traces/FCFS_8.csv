1,0,0
2,0,0
3,0,0
4,1,5
5,1,5
6,1,5
7,1,5
8,2,1
9,2,1
10,2,1
11,2,1
12,2,1
13,3,2
14,4,2
15,4,2
16,4,2
17,4,3
18,4,3
19,5,3
20,5,3
