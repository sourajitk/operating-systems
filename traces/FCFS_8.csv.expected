1,0
2,0
3,0
4,6
5,11
6,16
7,21
8,22
9,23
10,24
11,25
12,26
13,28
14,30
15,32
16,34
17,37
18,40
19,43
20,46
